apps/articles/routes.coffee

```diff
@@ -63,6 +65,34 @@ setupEmailSubscriptions = (user, article, cb) ->
           res.locals.sd.SECTION = section.toJSON()
           res.render 'section', section: section, articles: articles
 
+@teamChannel = (req, res, next) =>
+  new Channel(id: req.params.slug).fetch
```

Comment:
Might help clean things up to extract this down to line ~90 to the model layer, maybe like `channel.fetchWithArticles`.